src/MixedModels.jl

```diff
@@ -99,6 +100,8 @@ export @formula,
        statscholesky,
        std,
        stderror,
+       testdata,
+       testdatasets,
        updateL!,
```

Comment:
Let's not export this name -- it seems likely to collide with other packages and I think the explicit `MixedModels.testdata` is nice.